recipe/patches/0013-Added-patch-for-rules_python-on-ppc64le.patch

```diff
@@ -0,0 +1,188 @@
+From 1f2207ee401b22a3cfc95d962a47b8941b8d9b09 Mon Sep 17 00:00:00 2001
+From: Nishidha Panpaliya <npanpa23@in.ibm.com>
+Date: Wed, 28 Jun 2023 14:10:51 -0400
+Subject: [PATCH] Add support of linux ppc64le
+
+---
+ WORKSPACE                                     |  19 ++-
+ .../0001-Add-support-of-linux-ppc64le.patch   | 131 ++++++++++++++++++
+ 2 files changed, 143 insertions(+), 7 deletions(-)
+ create mode 100644 thirdparty/patches/0001-Add-support-of-linux-ppc64le.patch
+
+diff --git a/WORKSPACE b/WORKSPACE
+index eb6aeba907..e8d744a8e8 100644
+--- a/WORKSPACE
++++ b/WORKSPACE
+@@ -7,6 +7,18 @@ ray_deps_setup()
+ 
+ load("//bazel:ray_deps_build_all.bzl", "ray_deps_build_all")
+ 
++http_archive(
++    name = "rules_python",
++    sha256 = "94750828b18044533e98a129003b6a68001204038dc4749f40b195b24c38f49f",
++    strip_prefix = "rules_python-0.21.0",
++    url = "https://github.com/bazelbuild/rules_python/releases/download/0.21.0/rules_python-0.21.0.tar.gz",
++    patch_args = ["-p1"],
++    patches = [
++       "@com_github_ray_project_ray//thirdparty/patches:0001-Add-support-of-linux-ppc64le.patch",
++    ],
++)
++
++
+ ray_deps_build_all()
+ 
+ # This needs to be run after grpc_deps() in ray_deps_build_all() to make
+@@ -31,13 +43,6 @@ load("@hedron_compile_commands//:workspace_setup.bzl", "hedron_compile_commands_
+ 
+ hedron_compile_commands_setup()
+ 
+-http_archive(
+-    name = "rules_python",
+-    sha256 = "94750828b18044533e98a129003b6a68001204038dc4749f40b195b24c38f49f",
+-    strip_prefix = "rules_python-0.21.0",
+-    url = "https://github.com/bazelbuild/rules_python/releases/download/0.21.0/rules_python-0.21.0.tar.gz",
+-)
+-
+ load("@rules_python//python:repositories.bzl", "python_register_toolchains")
+ 
+ python_register_toolchains(
+diff --git a/thirdparty/patches/0001-Add-support-of-linux-ppc64le.patch b/thirdparty/patches/0001-Add-support-of-linux-ppc64le.patch
+new file mode 100644
+index 0000000000..40822a2203
+--- /dev/null
++++ b/thirdparty/patches/0001-Add-support-of-linux-ppc64le.patch
+@@ -0,0 +1,131 @@
++From 6390b409b608cb8d295ae667c7d531ee8fdf92b0 Mon Sep 17 00:00:00 2001
++From: Nishidha Panpaliya <npanpa23@in.ibm.com>
++Date: Wed, 28 Jun 2023 14:10:00 -0400
++Subject: [PATCH] Add support of linux ppc64le
++
++---
++ python/versions.bzl | 56 +++++++++++++++++++++++++++++++++++++++++----
++ version.bzl         |  2 +-
++ 2 files changed, 52 insertions(+), 6 deletions(-)
++
++diff --git a/python/versions.bzl b/python/versions.bzl
++index 662f89d..1008084 100644
++--- a/python/versions.bzl
+++++ b/python/versions.bzl
++@@ -86,6 +86,17 @@ TOOL_VERSIONS = {
++         },
++         "strip_prefix": "python",
++     },
+++    "3.8.16": {
+++        "url": "20230507/cpython-{python_version}+20230507-{platform}-{build}.tar.gz",
+++        "sha256": {
+++            "aarch64-apple-darwin": "1e0a92d1a4f5e6d4a99f86b1cbf9773d703fe7fd032590f3e9c285c7a5eeb00a",
```

Comment:
Do we need to include all the values from the platforms we don't support anyway?

Reply:
Well, I've not added the real sha values, those are just copy of above ones. I've added in case it fails for not finding anything (like syntactic check).